recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,34 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: '_storageDurability',
+  DURABILITY_KEY: 'durable',
+  async seedDurability(sandbox) {
+    const globalDurabilityStore = Storage.makeStorage(this.DURABILITY_NAMESPACE, sandbox);
```

Comment:
Technically, if an action tried to `makeStorage('_storageDurability')` then yes - the data would collide. In my opinion I don't think that's a huge thing to worry about - but I could be wrong. Should we specify it further, something like `normandy_storageDurability`?

While I'm on the subject - I think the term `prefix` should be swapped to `namespace` in  the storage code. Though we use prefixes within the actual `Storage`s, to outside modules the term `prefix` is a little vague. It makes sense once you open up the Storage file and see that prefixes effectively namespace the data, but at first the terminology didn't click for me. 

Also, if we switch to the term `namespace`, outside modules won't have to be updated if we ever decide to change how `Storage`s internally handle the data. Working with stores then would become a little less coupled.

Just a thought!